src/Error/ErrorHandler.php

```diff
@@ -97,6 +97,7 @@ public function __construct($options = [])
             'log' => true,
             'trace' => false,
             'exceptionRenderer' => ExceptionRenderer::class,
+            'displayDeprecations' => false,
```

Comment:
with auto switching deprecation warnings into debug_kit , we can set `True` here

Reply:
I'm trying to make it easier for plugin authors by not failing all their tests by default. Setting this to true by default would not address that.